pennylane/ops/op_math/controlled_class.py

```diff
@@ -265,6 +265,9 @@ def _wires(self, new_wires):
 
     # Methods ##########################################
 
+    def __repr__(self):
+        return f"Controlled({self.base}, control_wires={self.control_wires.tolist()})"
```

Comment:
Since `repr` is supposed to include *all* relevant information, maybe we should include `work_wires` and `control_values` if they are non-trivial.